### DISCUSSION TITLE
Rename function input arguments to rename docstring Args automatically

### DISCUSSION THREAD

**Discussion by amir-abdi** [microsoft/pylance-release#3111] [Enhancement]
When Renaming function input arguments, it is expected of the Args section of the docstring to get updated automatically. 
The Docstring style I tried was [Google-style](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html). 

This feature is available in PyCharm.

**Comment by bschnurr** [comment]
There are multiple docstring styles. was there one in particular you use?

**Reply by amir-abdi** [reply]
Google-style; description updated.

**Reply by haoxiangsnr** [reply]
Google-style, me too.

**Comment by sudoandros** [comment]
Hi everyone. Seems like this discussion died down for some reason. I would like this functionality, too. And I see I'm not alone. Are there any plans to support it? I see that Pyright already is parsing the docstring for the means of proper documentation popup. Is it  maybe possible to use the same data to "bind" symbols to their docstring counterparts? As was said earlier, Google style docs would be a good start. It would be a great addition which allows developers to better avoid mistakes in the project's API documentation.

**Reply by rchiodo** [reply]
> Are there any plans to support it?

Generally, we use the upvotes on discussions to decide what to work on next. You can see our ranking here:
https://github.com/microsoft/pylance-release/discussions?discussions_q=sort%3Atop+is%3Aopen
